base/bitarray.jl

```diff
@@ -1296,17 +1296,12 @@ function (&)(B::BitArray, x::Bool)
 end
 (&)(x::Bool, B::BitArray) = B & x
 
-function (|)(B::BitArray, x::Bool)
-    x ? trues(size(B)) : copy(B)
```

Comment:
This will lead to a performance regression, since the corresponding .| version will have to check each element?